testutil/checkers_test.go

```diff
@@ -36,8 +36,7 @@ import (
 	"testing"
 
 	"gopkg.in/check.v1"
-
-	"github.com/snapcore/snapd/strutil"
+	// "github.com/snapcore/snapd/strutil"
```

Comment:
Why is this commented out? If we don't need it we can just remove it I guess?

Reply:
Uhh, I'll force push to drop this one.